frontend/src/pages/admin/Admins/components/AdminListItems.js

```diff
@@ -0,0 +1,32 @@
+import { Avatar, List, Typography } from "antd";
+import { blue } from "@ant-design/colors";
+
+const { Text } = Typography;
+
+const AdminListItems = ({ admin, isCurrentUser }) => {
+  const you = isCurrentUser ? (
+    <Text keyboard type="warning">
+      You
+    </Text>
+  ) : (
+    ""
+  );
```

Comment:
- you may use default null to avoid `else` block.
- actually no need to declare `you` you can insert this to the component directy.
```suggestion
const AdminListItems = ({ admin, isCurrentUser = null }) => {
  const you = isCurrentUser && (
    <Text keyboard type="warning">
      You
    </Text>
  )
```